tests/test_rsa_keys.py

```diff
@@ -27,6 +27,7 @@
 
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.primitives.hashes import SHA256
```

Comment:
Unused constant? We really need a linter.

Reply:
Good catch

Reply:
What's a "flag day"?

Reply:
Ultimately, I think a linter (and using a standard coding style) makes the code easier to work on so I am in favour of the flag day (event where the entire codebase changes to the new style, making a large diff with only mechanical changes which invalidate patches and make revision control "noise").